src/app/app.component.html

```diff
@@ -22,4 +22,12 @@
         </mat-toolbar>
         <router-outlet></router-outlet> 
     </div>
-</div>
+    <div *ngIf="isLightTheme$ | async; else darkVersion" class="footer"> 
+        <p>Version {{ getVersion() }}<p>
+    </div>
+    <ng-template #darkVersion>
+        <div class="footer"> 
+            <p style="color: white">Version {{ getVersion() }}<p>
```

Comment:
convention: all styling should be done in scss files. Classes should then be applied to HTML tags

Reply:
Got it. I will use a span tag instead.